easybuild/easyconfigs/s/SUPPA/SUPPA-2.3-20231005-foss-2022b.eb

```diff
@@ -0,0 +1,41 @@
+easyblock = 'Tarball'
+
+name = 'SUPPA'
+version = '2.3-20231005'
+local_commit = 'a5d5e89'
+
+homepage = 'https://github.com/comprna/SUPPA'
+description = """Fast, accurate, and uncertainty-aware differential
+ splicing analysis across multiple conditions."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+
+github_account = 'comprna'
+source_urls = [GITHUB_SOURCE]
+sources = ['%s.tar.gz' % local_commit]
+checksums = ['5e55911be2727c7abc7a4e3b6179f2f7cb0e3126bb443bce102fa7af97249e5d']
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('SciPy-bundle', '2023.02'),
+    ('statsmodels', '0.14.0'),
+    ('scikit-learn', '1.2.1'),
+]
+
+postinstallcmds = ['cd %(installdir)s && mkdir bin && ln -s suppa.py bin/suppa.py && chmod a+x suppa.py']
```

Comment:
symlink is not correct here, the `suppa.py` in `bin` should point to `../suppa.py`, not `suppa.py`

if you create the symlink correctly, please check for `bin/suppa.py` in `sanity_check_files`, and remove the `PATH` update in `modextrapaths` (no longer needed, EB will automatically add `bin` to `$PATH`)

Reply:
or just remove the symlink, it's not worth it imo.

Reply:
(to fix the symlink, add option `-r` to `ln`)

Reply:
fixed in new PR https://github.com/easybuilders/easybuild-easyconfigs/pull/19275